### ISSUE TITLE
code error

### ISSUE THREAD

**Issue by XiaoBin2001** [noodle-lab/GaussianSpa#3]
gaussians.prune_points(mask)在使用spa方法时出现了两次，应删除在print之前的那一次。

            if iteration == args.optimizing_spa_stop_iter:
                imp_score = update_imp_score( scene, args, gaussians, pipe, background)
                k_threshold = int((1-opt.prune_ratio2) * imp_score.shape[0])
                _, indices = torch.topk(imp_score, k=k_threshold, largest=True)
                mask = torch.ones(imp_score.shape[0], dtype=bool)
                mask[indices] = False
                gaussians.prune_points(mask)
                print("\nBefore sparsifyting pruning:",len(gaussians.get_opacity)) 
                gaussians.prune_points(mask)
                print("\nAfter sparsifying pruning",len(gaussians.get_opacity))
                torch.cuda.empty_cache()

**Comment by zhangseammm** [comment]
Sorry for that. We have corrected it.
